README.md

```diff
@@ -48,4 +50,49 @@ openvpn_server_client_sent_bytes_total{common_name="...",connection_time="...",r
 openvpn_server_route_last_reference_time_seconds{common_name="...",real_address="...",status_path="...",virtual_address="..."} 1.493018841e+09
 openvpn_status_update_time_seconds{status_path="..."} 1.490089154e+09
 openvpn_up{status_path="..."} 1
+openvpn_connected_clients 1
+```
+
+## Usage
+
+Usage of openvpn_exporter:
+```
+  -openvpn.status_paths string
+    	Paths at which OpenVPN places its status files. (default "examples/client.status,examples/server2.status,examples/server3.status")
+  -web.listen-address string
+    	Address to listen on for web interface and telemetry. (default ":9176")
+  -web.telemetry-path string
+    	Path under which to expose metrics. (default "/metrics")
+```
+
+E.g:
+```
+openvpn_exporter -openvpn.status_paths /etc/openvpn/openvpn-status.log
+```
+
+## Docker
+
+Build the image:
+```
+docker build --force-rm=true -t openvpn_exporter .
+```
+
+The final image is around 8MB. A temporary image has been downloaded(Golang) to make the final one. Once built, this temporary image become orphan, you can delete it:
+```
+docker rmi -f $(docker images | grep "<none>" | awk "{print \$3}")
+```
+
+To use with docker you must mount your status file to `/etc/openvpn_exporter/server.status`.
+```
+docker run -it -p 9176:9176 -v /path/to/openvpn_server.status:/etc/openvpn_exporter/server.status openvpn_exporter
+```
+
+Metrics should be available on your host IP: http://<host_ip>:9176/metrics. E.g: http://10.39.9.94:9176/metrics
+
+
+## Get a standalone executable binary
```

Comment:
We already had `build_static.sh` for this purpose. Should this script be reimplemented on top of this Dockerfile now? Maybe also worth mentioning it here in the documentation.

Reply:
Yes I saw that. I left it.
Here we can:
- delete the sh script and so redirect users to the docker run command I've added to get a static file
- doc the sh script and remove the part about getting a static file from docker command

Up to you.

Reply:
Deleting the shell script sounds fine by me, as long as it's documented.

Reply:
Done